### PR TITLE
Milestone 1 

### PR DESCRIPTION
**Pull request by Tchilo**
- Add .html .css .js files 
- Add linter files
- Update README.md
- Create welcome page, main page, and speaker page 
- Load featured speakers dynamically
- Add about page
- Add images of speakers 
- Deploy project

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      <li>
+        <img src="images/download.jpeg" alt="">
+        <div class="line-t"></div>
+        <div>
+          <h2>Daniel Dennett</h2>
+          <span>American philosopher</span>
+          <div class="break"></div>
+          <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ad dolores, voluptate.</p>
+        </div>
+
```
Comment (mikenath223) [inline index.html]: - [x] The speakers' section should be dynamically generated with js and appended to the DOM. Please include an array of objects each containing a speaker's info then loop through it to append to the DOM.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <h2 class="program-head fs">Featured Speakers</h2>
+    <div class="program-line red fs-li"></div>
+    <ul class="li-of-speakers">
+      <li>
+        <img src="images/download.jpeg" alt="">
+        <div class="line-t"></div>
+        <div>
+          <h2>Daniel Dennett</h2>
+          <span>American philosopher</span>
+          <div class="break"></div>
```
Comment (mikenath223) [inline index.html]: - [x] The speaker section is expected to resemble the given design kindly ensure you include the background image seen in the original.

| Original | Yours |
| --------- | ---------- |
|  |  |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+  <title>Consciousness Explained</title>
+</head>
+
```
Comment (mikenath223) [inline index.html:14]: - [x] Kindly include the about page ad requested in the project requirement.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+body {
+  background-image: url('images/bg-img.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+html {
+  scroll-behavior: smooth;
+}
```
Comment (mikenath223) [inline style.css]: - [x] The font-family is expected to be: Cocogoose, Lato. As seen in the given design please make the change.

File: index.html (hunk 2 above)
Comment (Tchilo) [reply index.html]: where can I get that image from?

Comment (mikenath223) [reply index.html]: You can create it with CSS. Here's an example:

File: index.html (hunk 1 above)
Comment (Tchilo) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (Tchilo) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (Tchilo) [reply index.html:14]: fixed

File: style.css (hunk 4 above)
Comment (Tchilo) [reply style.css]: fixed

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested in the previous review 👏 
- About page has been added.
- Home page is fixed.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+function grab(e) {
+  return document.querySelector(`.${e}`);
+}
+
```
Comment (mikenath223) [inline about.js]: - [x] Please change this to es6 arrow function syntax as requested in the JavaScript best practices.

File: main.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
+function grab(e, isId = false, qAll = false) {
+  if (isId) {
+    return document.getElementById(e);
+  } if (qAll) {
+    return document.querySelectorAll(`.${e}`);
+  }
+  return document.querySelector(`.${e}`);
+}
```
Comment (mikenath223) [inline main.js]: - [x] Kindly change this to es6 arrow function syntax also.

File: main.js [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    <h2>${el.Name}</h2>
+    <span>${el.job}</span>
+    <div class="break"></div>
+    <p>${el.text}</p>
+     </div>
+  </div>
+<li>`;
+
+  grab('more', false, false).addEventListener('click', () => {
+    // const check = grab('more', false, false);
```
Comment (mikenath223) [inline main.js]: - [x] Please remove the commented code as it makes the codebase quite untidy, you can refer to GitHistory if you want to refer to previous code versions.

Comment (Tchilo) [reply main.js]: fixed

File: main.js (hunk 6 above)
Comment (Tchilo) [reply main.js]: fixed

File: about.js (hunk 5 above)
Comment (Tchilo) [reply about.js]: fixed

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**